lib/active_admin/views/pages/index.rb

```diff
@@ -110,9 +110,10 @@ def find_index_renderer_class(klass)
         end
 
         def render_blank_slate
-          blank_slate_content = I18n.t("active_admin.blank_slate.content", resource_name: active_admin_config.plural_resource_label)
+          localizer = ActiveAdmin::Localizers.resource(active_admin_config)
+          blank_slate_content = localizer.t('blank_slate.content', resource_name: active_admin_config.plural_resource_label)
```

Comment:
Our i18n linter is saying that we're missing translations for this string. But the underlying problem seems that the string is incorrect? Isn't is missing the `activeadmin.` namespace?

Reply:
Did you mean that there should be like that: `localizer.t('active_admin.blank_slate.content', resource_name: active_admin_config.plural_resource_label)` ?